pkg/descheduler/descheduler.go

```diff
@@ -107,6 +109,15 @@ func Run(ctx context.Context, rs *options.DeschedulerServer) error {
 		return nil
 	}
 
+	// Add k8s compatibility warnings to logs
+	serverVersion, serverErr := rs.Client.Discovery().ServerVersion()
+	if serverErr != nil {
+		klog.V(1).InfoS("Warning: Get Kubernetes server version fail")
+		return serverErr
+	}
+	if descheduler_minor_version != serverVersion.Minor {
+		klog.V(1).InfoS(fmt.Sprintf("Warning: Descheduler minor version %s is not supported on your version of Kubernetes %s.%s. See compatibility docs for more info: https://github.com/kubernetes-sigs/descheduler#compatibility-matrix", descheduler_minor_version, serverVersion.Major, serverVersion.Minor))
```

Comment:
thoughts on klog.Warning?

Reply:
before i did want to use klog.warning, but i check the code here: https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/descheduler.go#L100
it still use `klog.V(1).InfoS("Warning: DryRun is set to True. You need to disable it to use Leader Election.")` , ..that's why..